packages/app-mobile/android/app/src/main/java/net/cozic/joplin/textinput/TextInputPackage.java

```diff
@@ -0,0 +1,38 @@
+package net.cozic.joplin.textinput;
+
+import android.text.Selection;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReadableArray;
+import com.facebook.react.uimanager.ViewManager;
+import com.facebook.react.views.textinput.ReactEditText;
+import com.facebook.react.views.textinput.ReactTextInputManager;
+
+import java.util.Collections;
+import java.util.List;
+
+public class TextInputPackage implements com.facebook.react.ReactPackage {
```

Comment:
@roman-r-m, could you please add some comments to this class explaining why it's needed and what it does? Then link to the relevant React Native issue, so that we know to remove the workaround once it's fixed.

Reply:
will do